src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Cohort/Draft.cshtml

```diff
@@ -0,0 +1,50 @@
+﻿@using SFA.DAS.Encoding
+@model SFA.DAS.EmployerCommitmentsV2.Web.Models.Cohort.DraftViewModel
+@inject IEncodingService EncodingService
+
+@{
+    ViewData["Title"] = "Draft";
+}
+
+<div class="govuk-grid-row">
+
+        <div class="govuk-grid-row">
+            <div class="govuk-width-container">
+                <h1 class="govuk-heading-xl">Draft apprentice details</h1>
+
+                <table class="govuk-table das-table--responsive">
+                    <thead class="govuk-table__head">
+                        <tr class="govuk-table__row">
+                            <th scope="col" class="govuk-table__header das-table-cell-width-25">Training provider</th>
+                            <th scope="col" class="govuk-table__header das-table-cell-width-25">Reference</th>
+                            <th scope="col" class="govuk-table__header das-table-cell-width-25 govuk-table__header--numeric">Number of apprentices</th>
+                            <th scope="col" class="govuk-table__header govuk-table__header--numeric das-table-cell-width-25"></th>
+                        </tr>
+                    </thead>
+                    <tbody class="govuk-table__body">
+
+                        @foreach (var cohort in Model.Cohorts)
+                        {
+                            var cohortReference = EncodingService.Encode(cohort.CohortId, EncodingType.CohortReference);
```

Comment:
We could encode the Id in the view model mapper just as easily, and reduce clutter